test/Repositories/BooksRepository.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { BookDTO } from 'App/Dto/BookDTO'
+import { Book } from 'App/Models/Book'
+import BooksRepository from 'App/Repositories/BooksRepository'
+import test from 'japa'
+
+test('BooksRepository', () => {
+  let book: BookDTO
+  let sut: BooksRepository
+  let booksAfterDeletion: Book[]
+
+  test.group('add', (group) => {
+    let bookBeforeDeletion: Book
+
+    group.before(() => {
+      book = {
```

Comment:
esse cara nõa é de fato `book`, e sim `params`... talvez faça mais sentido?

```suggestion
      params = {
```